pkgs/tools/graphics/zxing-cpp/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, opencv
+}:
+stdenv.mkDerivation rec {
+
+  pname = "zxing-cpp";
+  version = "0.0";
```

Comment:
Naming/Versioning of stuff without a version is currently discussed here:
https://github.com/NixOS/nixpkgs/pull/88023

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/88065/commits/23285b04ab7f0a19ae3532b8317b13773f649be6